### ISSUE TITLE
[Feature] Create Product Listing Page

### ISSUE THREAD

**Issue by ShwetKhatri2001** [ShwetKhatri2001/AR-Webstore#65]
### Description

We already have a `ModelViewer` component having a 3D viewer and QR code ( for desktop ).
Reuse the component and list it responsively for multiple products.
For having multiple products try to add some more freely available 3D models ( .glb and .usdz ).

### Screenshots

Just go through some similar e-commerce platforms to get an idea of how the UI should look.

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines]
- [ ] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

**Comment by Mrudula1205** [comment]
@ShwetKhatri2001 I would like to work on this issue. Please assign it to me

**Comment by ShwetKhatri2001** [comment]
Sure, You're assigned. Show your creativity and also try to complete it in 5 days.

**Comment by ShwetKhatri2001** [comment]
Any updates @Mrudula1205 ? Please try to complete your changes in the next 3 days.
After that, it will be open to other contributors.

**Comment by Mrudula1205** [comment]
I am working on it. I'll update soon.

**Comment by ShwetKhatri2001** [comment]
Any updates or issues @Mrudula1205 ? ( if you're still working on this issue )
Please try to make the requested changes and update your PR within the next 3 days ( if you want to ).
After that, This issue will be open to other interested contributors.

**Comment by Mrudula1205** [comment]
I'm working on it.

**Comment by ShwetKhatri2001** [comment]
This issue is open to make enhancements on the work done.

**Comment by iSubhamMani** [comment]
hey @ShwetKhatri2001 I would like to work on this issue. Just to clarify I'll have to add more products right ?

**Comment by ShwetKhatri2001** [comment]
Yes, you can add few more and mostly change the product card's UI. 
Look at the UI on the desktop and mobile both before making any changes. 
The changes you make should also be responsive for both.

**Comment by iSubhamMani** [comment]
Okay assign me the issue

**Comment by ShwetKhatri2001** [comment]
Any updates @iSubhamMani ? Please try to complete your changes in the next 3 days.

**Comment by iSubhamMani** [comment]
Hey @ShwetKhatri2001 Sorry but I'm unable to complete the task at the moment, I'm unassigning myself

**Comment by ShwetKhatri2001** [comment]
Please have a look at the requirements before you get it assigned the next time.
